CHANGELOG.md

```diff
@@ -7,7 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Please check our [developers guide](https://gitlab.com/tokend/developers-guide)
 for further information about branching and tagging conventions.
 
-## [Unrealised]
+## [Unreleased]
+#### Added
+- Modules trees for unverified users
+- `UnverifiedUserModuleItem` component
+
+#### Fixed
+- `ModuleTreesItem` component
+
+## [0.4.0] - 2023-07-05
```

Comment:
- how did you add tag to changelog and unreleased at the same time?
- you removed brackets on other tags and added brackets to this one